CMakeLists.txt

```diff
@@ -58,6 +102,8 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
 # Set compiler flags and options.
 if( MSVC )
+# set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /MTd")
+# set (CMAKE_LIB_LINKER_FLAGS "${CMAKE_LIB_LINKER_FLAGS} /NODEFAULTLIB:libucrt.lib")
 elseif( UNIX OR MINGW )
```

Comment:
Is this if else is necessary in this situation when windows code is commented out?